content/docs/05-ai-sdk-ui/21-error-handling.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Error Handling
+description: Learn how to handle errors in the AI SDK UI
+---
+
+# Error Handling
```

Comment:
There is also the "error" helper that is return and can be used for rendering

Reply:
added thanks!